src/components/common/needsShutdown.scss

```diff
@@ -0,0 +1,3 @@
+.configuration-changes-list {
```

Comment:
This should be better done by putting the two items in a Stack container with hasGutter. can be followup.